src/components/characters.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import "./style.css";
+import { Card } from "baseui/card";
+import { wrap } from "module";
+
+type CharacterCardProps = {
+  id: number;
+  name: string;
+  image: string;
+  onClick?: () => void;
+};
+
+const CharacterCard: React.FC<CharacterCardProps> = ({
+  id,
+  name,
+  image,
+  onClick,
+}) => {
+  return (
+    <div onClick={onClick}>
+      <Card
+        overrides={{
+          Root: {
+            style: {
+              display: "flex",
+              flexDirection: "row",
+              flexWrap: "wrap",
+              justifyContent: "center",
+              alignItems: "center",
+              width: "200px",
+              minHeight: "250px",
+              borderRadius: "10px",
+              margin: "40px",
+              textAlign: "center",
+              backgroundColor: "gray",
+              border: "2px solid",
+            },
```

Comment:
styles should be in a separate stylesheet or styles.ts file

Reply:
Noted

Reply:
Resolved